oak_containers_orchestrator/src/main.rs

```diff
@@ -94,6 +96,24 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         tokio::fs::create_dir_all(path).await?;
     }
 
+    let key_provisioning_role = launcher_client
+        .get_key_provisioning_role()
+        .await
+        .map_err(|error| anyhow!("couldn't get key provisioning role: {:?}", error))?;
+    let key_store = Arc::new(match key_provisioning_role {
+        KeyProvisioningRole::Unspecified => Err(anyhow!("unspecified key provisioning role"))?,
```

Comment:
I think `Err(anyhow!(...))?` can be simplified as `anyhow::bail!(...)`?

Reply:
Done